R/viz.R

```diff
@@ -2370,4 +2369,4 @@ plot_multi_timepoint = function(mafs,
     return(TRUE)
   }
   return(FALSE)
-}
+}
```

Comment:
I'm seeing some of these warnings about missing newlines. Sometimes these are real and sometimes github detects them erroneously but this one looks real. Can you make sure all files end in a newline?

Reply:
Yes, I did put that on my list of things to do!